invenio_orcid/config.py

```diff
@@ -26,6 +26,12 @@
 
 ORCID_SYNCHRONIZATION_ENABLED = True
 
+ORCID_AUTHORS_SEARCH_CLASS = "invenio_search:RecordsSearch"
+
+ORCID_RECORDS_PID_TYPE = 'literature'
```

Comment:
Actually would it be possible to make this a list? Already in INSPIRE we are going to push to ORCID `literature` and `data`.